<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkersWidget</class>
 <widget class="QWidget" name="WorkersWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>840</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FinanMemberWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#FinanMemberWidget{background-color: rgb(255, 255, 255);border-radius:8px;}</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>116</y>
     <width>780</width>
     <height>431</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>740</width>
     <height>310</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>780</width>
     <height>600</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget{border-bottom-left-radius:0px;}
QListWidget::Item{background:rgb(255,255,255);}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectItems</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>90</y>
     <width>27</width>
     <height>15</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>27</width>
     <height>15</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>270</width>
     <height>15</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14px &quot;微软雅黑&quot;;
color:#333333;
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>操作</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>90</y>
     <width>27</width>
     <height>15</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>27</width>
     <height>15</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>270</width>
     <height>15</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14px &quot;微软雅黑&quot;;
color:#333333;
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>状态</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>60</width>
     <height>15</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>27</width>
     <height>15</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>270</width>
     <height>15</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14px &quot;微软雅黑&quot;;
color:#333333;
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>工单编号</string>
   </property>
  </widget>
  <widget class="QWidget" name="TopWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>16</y>
     <width>840</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>52</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>840</width>
     <height>52</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(252, 252, 252);
border-bottom:1px solid rgb(222, 222, 222);
border-top-right-radius:8px;
border-top-left-radius:8px;</string>
   </property>
   <widget class="QLabel" name="labelTitle">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>15</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>18</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>18</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16px &quot;微软雅黑&quot;;
color: rgb(51, 51, 51);
border-bottom:0px;</string>
    </property>
    <property name="text">
     <string>我的工单</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>10</y>
      <width>32</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>32</width>
      <height>28</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>32</width>
      <height>28</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/EntCenterWidget/image/close.png);}
QPushButton:hover,pressed{border-image: url(:/EntCenterWidget/image/close_sel.png);};</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>60</width>
     <height>15</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>27</width>
     <height>15</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>270</width>
     <height>15</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14px &quot;微软雅黑&quot;;
color:#333333;
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>工单标题</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>90</y>
     <width>60</width>
     <height>15</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>27</width>
     <height>15</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>270</width>
     <height>15</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14px &quot;微软雅黑&quot;;
color:#333333;
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>创建时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="previousPageLabel">
   <property name="geometry">
    <rect>
     <x>401</x>
     <y>563</y>
     <width>16</width>
     <height>36</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>上一页</string>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>495</x>
     <y>563</y>
     <width>15</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>第</string>
   </property>
  </widget>
  <widget class="QLabel" name="leftSeparateLabel">
   <property name="geometry">
    <rect>
     <x>427</x>
     <y>563</y>
     <width>16</width>
     <height>36</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>下一页</string>
   </property>
   <property name="text">
    <string>..</string>
   </property>
  </widget>
  <widget class="QLabel" name="nextPageLabel">
   <property name="geometry">
    <rect>
     <x>479</x>
     <y>563</y>
     <width>16</width>
     <height>36</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>下一页</string>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>563</y>
     <width>15</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>页</string>
   </property>
  </widget>
  <widget class="QWidget" name="centerPagesWidget" native="true">
   <property name="geometry">
    <rect>
     <x>443</x>
     <y>563</y>
     <width>10</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="leftPagesWidget" native="true">
   <property name="geometry">
    <rect>
     <x>417</x>
     <y>563</y>
     <width>10</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="rightSeparateLabel">
   <property name="geometry">
    <rect>
     <x>453</x>
     <y>563</y>
     <width>16</width>
     <height>36</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>下一页</string>
   </property>
   <property name="text">
    <string>..</string>
   </property>
  </widget>
  <widget class="QWidget" name="rightPagesWidget" native="true">
   <property name="geometry">
    <rect>
     <x>469</x>
     <y>563</y>
     <width>10</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pageLineEdit">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>570</y>
     <width>50</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="EntCenterWidget.qrc"/>
 </resources>
 <connections/>
</ui>
